cmd/errors.go

```diff
@@ -29,3 +29,19 @@ with as many details & logs as you can share!
 
 ` + colorstring.Redf("Error: %s", e.msg)
 }
+
+// MissingCodesigningFilesError ...
+type MissingCodesigningFilesError struct {
```

Comment:
Do you need this type? Isn't an exported constant string is enough which can be appended to the current error?